features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature

```diff
@@ -0,0 +1,34 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats onbekend
+
+  Rule: In het antwoord geleverde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Alleen velden die echt geleverd zijn, dus een waarde hebben in de response, worden opgenomen als geleverde rubriek
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
+    
+    Abstract Scenario: Protocolleren van geleverde velden voor <fields veld> voor een persoon met onbekende verblijfplaats
+      Gegeven de persoon met burgerservicenummer '000000061' heeft de volgende gegevens
+      | naam            | waarde     |
+      | anummer (01.10) | 1041968780 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                   | waarde   |
+      | gemeente van inschrijving (09.10)      | 1999     |
+      | land (13.10)                           | 0000     |
+      | datum aanvang adres buitenland (13.20) | 00000000 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000061                       |
+      | fields              | <fields>                        |
+      Dan is protocollering opgeslagen met de volgende gegevens
+      | anummer    | request_geleverde_rubrieken |
```

Comment:
Ik mis hier de kolom voor 'request_zoek_rubrieken' en ik mis in dit feature een rule voor 'request_geleverde_rubrieken'.

Reply:
heb ik toegevoegd in de Dan stap. Is alleen opgenomen om te illustreren dat er meer zit in request_zoek_rubrieken dan in request_geleverde_rubrieken, en te controleren dat beide in dezelfde protocolleringsrecord komen.

De rule op request_zoek_rubrieken staat in protocollering-vraag.feature